core/src/main/java/org/elasticsearch/bootstrap/BootstrapCheck.java

```diff
@@ -500,6 +499,38 @@ public final boolean isSystemCheck() {
 
     }
 
+    /**
+     * Checks if the serial collector is in use. This collector is single-threaded and devastating
+     * for performance and should not be used for a server application like Elasticsearch.
+     */
+    static class UseSerialGCCheck implements BootstrapCheck.Check {
+
+        @Override
+        public boolean check() {
+            return getUseSerialGC().equals("true");
```

Comment:
Should this fail if the jvm returns a funny string like `"rodent"`? Should it log anything in that case? Should `JvmInfo`?